IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/Entities/IfcRelInterferesElements/Documentation.md

```diff
@@ -1,12 +1,25 @@
+* If the interference geometry is omitted then the interference is provided as a logical relationship. Under this circumstance, the connection point, curve, surface, or solid has to be recalculated by the receiving application.  
   
-The _RelatingElement_ and _RelatedElement_ define the two elements in the relationship, that may have different roles. This is controlled by the attribute _ImpliedOrder_.  
-  
-\* _ImpliedOrder_=TRUE\S\  The _RelatingElement_ constitutes the primary element of the interference relationship. If the interference is to be resolved by subtracting the overlapping part, it should be subtracted from the _RelatingElement_. The net result would be the _RelatingElement_ subtracted by the _InterferenceGeometry_. This would be the case in interference relationships where the _RelatedElement_ creates a void in the _RelatingElement_ dynamically.  
-\* _ImpliedOrder_=FALSE\S\  The _RelatingElement_ and _RelatedElement_ have no priority among each other. If the interference is to be resolved then no information about whether the _InterferenceGeometry_ should be subtracted from the _RelatingElement_ or thed _RelatedElement_ can be traced. This would be the case for clash detection results.  
-\* _ImpliedOrder_=UNKNOWN No information about the priorities is provided.  
+The _RelatingElement_ and _RelatedElement_ define the two elements in the relationship, that may have different roles. The relation orientation may be forcefully required by some _InterferenceType_ values, this is done  by setting the attribute _ImpliedOrder_ accordingly:
   
+* _ImpliedOrder_=TRUE\S\  The _RelatingElement_ constitutes the primary element of the interference relationship that is oriented from _RelatingElement_ (source) to _RelatedElement_ (target). If the interference is to be resolved by subtracting the overlapping part, it should be subtracted from the _RelatingElement_. The net result would be the _RelatingElement_ subtracted by the _InterferenceGeometry_. This would be the case in interference relationships where the _RelatedElement_ creates a void in the _RelatingElement_ dynamically.  
+* _ImpliedOrder_=FALSE\S\  The _RelatingElement_ and _RelatedElement_ have no priority among each other. If the interference is to be resolved then no information about whether the _InterferenceGeometry_ should be subtracted from the _RelatingElement_ or the _RelatedElement_ can be traced. This would be the case for clash detection results.  
+* _ImpliedOrder_=UNKNOWN No information about the priorities is provided.  
+ 
+The _IfcConnectionGeometry_ property is used to define the interference shape, it can be part of the shape of one of the elements of the relationship (case of clash, overlapping and crossing relations) or be at the frontier of these shape (case on along, over and under relations).
```

Comment:
Suggestion: "(in the case of clash, overlapping and crossing interference types) or define the interface between the two shapes (in the case of along, over and under interference types)."

Reply:
Better English taken :smile: